cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java

```diff
@@ -358,21 +359,33 @@ private static Path validateBalaFile(BuildProject project, Path customBalaPath)
     }
 
     private static void validatePackageMdAndBalToml(Path balaPath) {
+        // Gets package name from balapath
+        String packageName = getPackageNameFromBalaName(balaPath.toFile().getName());
+
         try (ZipInputStream zip = new ZipInputStream(Files.newInputStream(balaPath, StandardOpenOption.READ))) {
             ZipEntry entry;
             while ((entry = zip.getNextEntry()) != null) {
+                // Checks if Package.md or README.md exists and is not empty
                 if (entry.getName().equals(
                         ProjectConstants.BALA_DOCS_DIR + "/" + ProjectConstants.PACKAGE_MD_FILE_NAME)) {
                     if (entry.getSize() == 0) {
                         throw new ProjectException(ProjectConstants.PACKAGE_MD_FILE_NAME + " cannot be empty.");
                     }
                     return;
+                } else if (entry.getName().equals(
+                        ProjectConstants.BALA_DOCS_DIR + "/" + ProjectConstants.MODULES_ROOT + "/"
```

Comment:
Instead of `/`, we can use `File.pathseparator()` which will behave according to the OS.

Reply:
Discussed with @azinneera and decided to use "/".

Reply:
In a ZIP file, the path is read in the UNIX pattern. Therefore, we cannot use the pathseparator.